docs/updater/update_process.md

```diff
@@ -158,3 +159,12 @@ if __name__ == '__main__':
     state = do_something(data)
     send_state(state)
 ```
+
+
+# Forced Updates
+
+A forced update overrides the normal update process to address situations where the authentication repository or its target repositories have encountered inconsistencies that prevent a standard update. During a forced update, the updater bypasses certain checks that would normally stop the process, such as the presence of an invalid or missing last validated commit. The forced update ensures that the repository is aligned with the latest commit found on the remote repository. 
+
+By using the --force flag, the updater will reset the state of the repository, discarding any uncommitted changes, and align the repository with the latest commit from the remote repository. This includes resetting the last validated commit to the latest commit found on the remote, and synchronizing the target repositories accordingly. If the target repositories have diverged or contain unaligned commits, the updater will reset them to the validated state, potentially discarding local changes.
```

Comment:
The updater should not set the last validated commit to a commit with might have not validated yet. The reason why we are doing so in a test is to simulate a case when commits were removed from the local repository manually, or when someone manually updated the last validated commit. Force flag can remove certain commits and uncommitted charges